client/product_details/Testing/endToEndTests.test.js

```diff
@@ -1,6 +1,6 @@
 import puppeteer from 'puppeteer';
 import { spawn } from 'child_process';
-import { exception } from 'console';
+import { last } from 'lodash';
```

Comment:
This is cool, I've never used it before!

Reply:
oh gosh! the last import statement was an autocorrect. I will remove that...